docker-compose.yaml

```diff
@@ -138,6 +138,7 @@ services:
       - WEBAPP_URL=${WEBAPP_URL}
       - WORKER_ENVIRONMENT=${WORKER_ENVIRONMENT}
       - WORKSPACE_ROOT=${WORKSPACE_ROOT}
+      - GITHUB_STORE_BRANCH=${GITHUB_STORE_BRANCH}
```

Comment:
Randomly landed here: if this is needed for cloud, we should probably update kustomize and helm charts as well.

Reply:
It's not needed for ☁️!